packet/templates/include/head.html

```diff
@@ -34,6 +34,35 @@
 
     <link rel="stylesheet" href="{{ url_for('static', filename='css/packet.min.css') }}">
 
+    <!-- Push Notifications -->
+    <script src="https://cdn.onesignal.com/sdks/OneSignalSDK.js" async=""></script>
+    <script>
```

Comment:
As much as possible from this will be moved into a `notifications.js` file in Phase 2